src/services/tree-sitter/index.ts

```diff
@@ -148,12 +180,20 @@ async function parseFile(
 
 			// Add separator if there's a gap between captures
 			if (lastLine !== -1 && startLine > lastLine + 1) {
-				formattedOutput += "|----\n"
+				if (addLineNumbers) {
+					formattedOutput += "||    ||----\n"
+				} else {
+					formattedOutput += "|----\n"
+				}
 			}
 			// Only add the first line of the definition
 			// query captures includes the definition name and the definition implementation, but we only want the name (I found discrepencies in the naming structure for various languages, i.e. javascript names would be 'name' and typescript names would be 'name.definition)
 			if (name.includes("name") && lines[startLine]) {
-				formattedOutput += `│${lines[startLine]}\n`
+				if (addLineNumbers) {
+					formattedOutput += `│| ${startLine} ||${lines[startLine]}\n`
```

Comment:
I think this will be even better if you include the end line so it knows exactly what to read in

```suggestion
					formattedOutput += `│| ${startLine} - ${endLine} ||${lines[startLine]}\n`
```

Reply:
Er... this just adds the same line number twice, so that's not exactly right. Is there some way to get the end line of the method though?